gamelogic/game.go

```diff
@@ -77,7 +91,14 @@ func (game *Game) Start() error {
 
 	game.inProgress = true
 
-	roleOrderAssigment := rand.Perm(len(game.rolePool))
+	var roleOrderAssigment []int
+	if os.Getenv("DEBUG_ROLE_ORDER") == "" {
```

Comment:
whati s all this for?

Reply:
it's so you can set an environment variable `export DEBUG_ROLE_ORDER="0,1,2,3,4,5"` instead of getting a random ordering every time. Convenient when debugging when i want to create a game and ensure that there's only one werewolf in the game (to test center choosing). In the future when we have more roles it will be convenient to enforce thigns like seer is player 0. That way you don't have to spam create game until you get the roles you want